versions/3.1.0.md

```diff
@@ -2336,6 +2336,20 @@ As such, inline schema definitions, which do not have a given id, *cannot* be us
 The [xml](#schemaXml) property allows extra definitions when translating the JSON definition to XML.
 The [XML Object](#xmlObject) contains additional information about the available options.
 
+###### Picking Schema Vocabularies
+
+OpenAPI documents are made up of multiple resources. Sometimes resources are all in the same root OpenAPI document (inline resources); also, resources can contain their own documents with `$ref` (separate resources). A schema object could exist as an inline resource, or a separate resource. 
```

Comment:
The opening sentence sounds like a re-visit of the `oasDocument` definition further up:

> A self-contained or composite resource which defines or describes an API or elements of an API.

We probably don't want two places saying nearly the same thing in different terms.

Reply:
The use of "inline" to mean within the same root OAS document conflicts with our use elsewhere (e.g. in the `discriminator` section) where "inline" means "not accessed via a `$ref` (internal or external)".

Reply:
Rather than say "sometimes this can happen" just say "this MAY happen".

Reply:
Do you actually mean "resources can **contain** their own **documents** with `$ref` (separate resources)"?

Reply:
Please can we avoid "could" and "can" along with "sometimes". Stick to "MAY" and other RFC2119 terms.

Reply:
This paragraph was setting up some stuff to make the following clear, but now that the following is simplified this can go away.